modules/apps/view-count/view-count-test/src/testIntegration/java/com/liferay/view/count/service/impl/test/ViewCountEntryLocalServiceTest.java

```diff
@@ -190,7 +236,14 @@ private Object _createSessionProxy(
 	@Inject
 	private static ViewCountEntryLocalService _viewCountEntryLocalService;
 
+	private final CyclicBarrier _cyclicBarrier = new CyclicBarrier(2);
+	private DB _db;
+	private SessionFactory _sessionFactory;
+	private final List<ViewCountEntry> _viewCountEntries = new ArrayList<>();
```

Comment:
@dantewang @yuhai _viewCountEntries is going to be used by two threads at the same time, it should be a thread safe collection.